index.html

```diff
@@ -13,7 +13,8 @@
        <header>
           <nav>
             <a class="logo" href="#">Tshepo45</a>
-       <div class="home-icons">
+            <a href="#"> <img class="img"  src="image/Union.png" alt="Union"/></a>
+            <div class="home-icons">
             <ul>
```

Comment:
- [ ] Good job putting effort into the project 👏 . However based on project requirements, you're expected to maintain the mobile view while making changes to the desktop version. you can see from the comparison below, that there's a margin space on the right relative to your screen. Kindly fix this to compliment the good job you've done so far, and give a better user experience 

## View:
![image](https://user-images.githubusercontent.com/50990925/222007336-10c0b91f-b74f-496e-a2f7-35580a4276e9.png)